### PR TITLE
Add Titouan in the core team

### PR DESCRIPTION
**Pull request by fabpot**

### CODE REVIEW

**Review by chalasr — APPROVED** [review]

**Review by sroze — APPROVED** [review]
Great job 🌟

**Review by Nyholm — APPROVED** [review]
Congratulations!

**Review by jderusse — APPROVED** [review]

File: contributing/code/core_team.rst [hunk 1]
```diff
@@ -72,6 +72,7 @@ Active Core Members
   * **Wouter De Jong** (`wouterj`_);
   * **Alexander M. Turek** (`derrabus`_);
   * **Jérémy Derussé** (`jderusse`_).
```
Comment (xabbuh) [inline contributing/code/core_team.rst]:
```suggestion
  * **Jérémy Derussé** (`jderusse`_);
```

**Review by xabbuh — APPROVED** [review]

**Comment by wouterj** [comment]
Welcome @tgalopin! Looking forward to see what Symfony UX will bring in the future!

**Pull request merged**